language/language-tests/copy.rs

```diff
@@ -0,0 +1,18 @@
+use hacspec_lib::*;
+
+#[derive(Clone, Copy)]
+pub struct MyStruct(usize);
+
+fn other(x: MyStruct) -> MyStruct {
+    x
+}
+
+pub fn first() -> MyStruct {
+    let s = MyStruct(0);
+    // Consuming s or rather copying it.
+    let s_copy = other(s);
```

Comment:
Normally you don't have to put `other(s)` here, just `s` is sufficient to move it.

Reply:
yeah, it's just to model real world usage.